config/samples/resources/computeforwardingrule/global-internal-forwarding-rule-with-target-http-proxy/compute_v1beta1_computeaddress.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeAddress
+metadata:
+  name: computeforwardingrule-dep-global-internal-app
```

Comment:
Should this be `computeforwardingrule-dep-global-internal-forwarding-rule-with-target-http-proxy`? Same question for the other sample files in this PR.

Reply:
changed to `computeforwardingrule-dep-global-internal-http-proxy` to stay within the name character limit.